### ISSUE TITLE
Option to disable legendart actions/resistances

### ISSUE THREAD

**Issue by ChrisPJohn** [ChrisPJohn/SolastaCommunityExpansion#117]
NicotineCaffeine

I hope someone could make an option for disabling "legendary" bullshit. I mean, this game for some things is already a pain in the arse, but with that 'add-on' it's simply terrible.

**Comment by DubhHerder** [comment]
If this option is implemented in the future, i suggest adding a warning of how removing legendary actions/resistances would severely nerf/unbalance every boss fight in the game as legendary actions aren't "add-ons" but are critical requirements in the action economy of single monster fights. Then it people will be able to make an informed choice either way.

for implementation, maybe something like:

   MonsterDefinition[] monster_definitions = DatabaseRepository.GetDatabase<MonsterDefinition>().GetAllElements();
       foreach (MonsterDefinition monster_definition in monster_definitions)
       {
       monster_definition.SetLegendaryCreature(false);    
       }

**Comment by ThyWoof** [comment]
I'm not a big fan of such a Nerf. It's really low priority for me as there are still level 20 features, a possible Warlock class, etc.

**Comment by ChrisPJohn** [comment]
I agree it's a low priority feature for me to work on personally. I'm trying to include ideas from the community, and since we have everything as disabled by default, it's ok to include features that aren't for everybody or that unbalance the game somewhat.

That said we don't need to include every possible idea in the mod, but I'm trying to track feature requests somewhere so that people feel heard and are encouraged to give feedback. Actual implementation of requests is going to of course be dependent on a modder wanting to tackle the work. Additionally for any feature we certainly should be aware of how it might break other aspects of the game. If something will break other features, we probably shouldn't do it. If something drastically changes the game balance like this but doesn't actually break anything I think it's ok to give as an option.
